### PR TITLE
First test for Discover-api

### PR DESCRIPTION
**Pull request by islamov089**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ coverage/
 node_modules/
 package-lock.json
 tsconfig.json
```
Comment (jkalandarov) [inline .gitignore:4]: tsconfig.json should be shared with everyone in the project

File: discoverApi.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  release_date: string;
+  title: string;
+  video: boolean;
+  vote_average: number;
+  vote_count: number;
+}
+
+async function fetchData(): Promise<AxiosResponse<MovieData>> {
+  const url =
+    'https://api.themoviedb.org/3/discover/movie?include_adult=false&include_video=false&language=en-US&page=1&sort_by=popularity.desc';
```
Comment (jkalandarov) [inline discoverApi.ts:27]: Are you always going to send requests to the base URL with the same query strings?

File: discoverApi.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  const options = {
+    headers: {
+      accept: 'application/json',
+      Authorization:
+        `Bearer ${apiKey}`,
+    },
+  };
+
+  try {
+    const response: AxiosResponse<MovieData> = await axios.get(url, options);
```
Comment (jkalandarov) [inline discoverApi.ts:37]: Are you always going to send GET request?

File: discoverApi.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    return response;
+  } catch (error) {
+    console.error('Error fetching data:', error);
+    throw error;
+  }
+}
+
+fetchData().then((data) => console.log(data));
```
Comment (jkalandarov) [inline discoverApi.ts:45]: Why are you calling the function right after you declared it?

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (islamov089) [reply .gitignore:4]: ok

File: discoverApi.ts (hunk 4 above)
Comment (islamov089) [reply discoverApi.ts:45]: sorry, I forgot to delete
I just wanted to see what was showing on the console.

**Pull request closed without merging**